transport/transport.go

```diff
@@ -63,13 +63,11 @@ type recvMsg struct {
 	err error
 }
 
-func (recvMsg) isItem() bool {
-	return true
-}
+func (*recvMsg) isItem() {}
 
 // All items in an out of a recvBuffer should be the same type.
 type item interface {
-	isItem() bool
```

Comment:
@tamird If this is a marker method, I feel so. There are some code in stdlib using similar approach (https://golang.org/src/go/ast/ast.go: type Expr interface is an example)